### PR TITLE
remove all occurrences of harmony & move @river to @river-build [HNT-7213]

### PR DESCRIPTION
**Pull request by miguel-nascimento**

### CODE REVIEW

**Review by texuf — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -32,30 +32,25 @@
...
-        "sampleapp:build": "turbo build --filter sample-app",
         "syncpack:check": "syncpack list-mismatches",
         "syncpack:fix": "syncpack fix-mismatches && syncpack format",
         "test": "turbo test",
-        "test:react": "vitest --project @river-build/react-sdk",
-        "test:build": "turbo test:build",
         "test:unit": "turbo test:unit -- --silent",
-        "worker:wait:stackup": "wait-on tcp:127.0.0.1:${PORT:-8686} --timeout=900000 --i=5000 --verbose",
-        "workers:build": "turbo build --filter gateway-worker --filter siwe-worker --filter token-worker --filter unfurl-worker --filter jwt-worker"
+        "test:react": "vitest --project @river-build/react-sdk",
```
Comment (texuf) [inline package.json:52]: this should be "test:react-sdk"

Comment (texuf) [reply package.json:52]: and second thought, i don't think we normally put the individual package tests at this level - can you just make sure that these tests run in the test:unit ci job? (I think we can add a test:unit to the react-sdk package that calls yarn test there.

**Review by texuf — APPROVED** [review]
didn't you make the harmony changes in a seperate pr? it's fine. 
ping me when you want to land this and we can run river_pull together.

File: package.json (hunk 1 above)
Comment (miguel-nascimento) [reply package.json:52]: I'll address this in another PR -- I think it makes sense to drop vitest for now, need to change the react-sdk package to use jest.

**Pull request merged**